scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -430,16 +431,17 @@ class Framework extends SbtFramework {
           annotationClass.getName
         else
           value
-      }
+      }).asInstanceOf[Array[String | Null]]
     
-    def execute(eventHandler: EventHandler, loggers: Array[Logger]) = {
+    def execute(eventHandler: EventHandler | Null, _loggers: Array[Logger | Null] | Null) = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.